packages/@glimmer/manager/lib/util/args-proxy.ts

```diff
@@ -6,10 +6,18 @@ import {
   CapturedPositionalArguments,
 } from '@glimmer/interfaces';
 import { Reference, valueForRef } from '@glimmer/reference';
-import { HAS_NATIVE_PROXY, enumerableSymbol } from '@glimmer/util';
+import { HAS_NATIVE_PROXY } from '@glimmer/util';
 import { Tag, track } from '@glimmer/validator';
 
-export const CUSTOM_TAG_FOR = enumerableSymbol('CUSTOM_TAG_FOR');
+const CUSTOM_TAG_FOR = new WeakMap<object, (key: string) => Tag>();
+
+export function getCustomTagFor(obj: object): ((key: string) => Tag) | undefined {
```

Comment:
Who uses this?

Reply:
We can avoid requiring the function to be bound eagerly by passing the object as the first arg. Not sure it matters too much though.

Reply:
👍  good suggestion, will do!